form.js

```diff
@@ -0,0 +1,21 @@
+const containerform = document.querySelector('.input');
+const error = document.querySelector('#error');
+containerform.addEventListener('submit', (e) => {
```

Comment:
- [ ] If a user submits the contact form with the email field containing uppercase characters an error message should show up, please implement this functionality it's a requirement